cli/src/forge.rs

```diff
@@ -128,6 +132,72 @@ fn main() -> eyre::Result<()> {
         Subcommands::Snapshot(cmd) => {
             cmd.run()?;
         }
+        Subcommands::Fmt { path, root, check } => {
+            if path.is_some() && root.is_some() {
+                return Err(eyre::eyre!("Only --root or path argument should be specified"))
+            }
+
+            let root = if let Some(path) = path {
+                path
+            } else {
+                let root = root.unwrap_or_else(|| {
+                    std::env::current_dir().expect("failed to get current directory")
+                });
+                if !root.is_dir() {
+                    return Err(eyre::eyre!("Root path should be a directory"))
+                }
+
+                ProjectPathsConfig::find_source_dir(&root)
+            };
+
+            let paths = if root.is_dir() {
+                WalkDir::new(root)
+                    .follow_links(true)
+                    .into_iter()
+                    .filter_map(|e| e.ok())
+                    .filter(|e| e.file_name().to_string_lossy().ends_with(".sol"))
+                    .map(|e| e.into_path())
+                    .collect()
```

Comment:
I think that's what `ethers_solc::utils::source_files` does?

Reply:
yep, fixed